api.py

```diff
@@ -473,11 +480,7 @@ def event_stream_gfw_data_api(
     else:
         raise ValueError(f"Invalid query type from frontend: {query_type}")
 
-    # Stores the stream to later write to the database
-    messages_store = []
```

Comment:
@batpad the linter was not happy about this line, as this var was never used. I guess the `messages_to_store` one below is the real variable being used.

Reply:
Oops! So sorry @yellowcap - getting used to the flow of this code-base - I did not run the linter locally. Thank you so much for fixing these.